### PR TITLE
chore: update README

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

with polished content per our latest update

## Testing

N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,14 +23,18 @@
...
+
+- A frontend-to-backend identity solution.
+  - A delightful sign-in experience for end-users and an OIDC-based identity service.
+  - Multi-platform SDKs that can integrate your apps with Logto quickly.
+- Out-of-box technology and UI support for many things you needed to code before.
+  - A centralized place to customize the user interface and then LIVE PREVIEW the changes you make.
+  - Social sign-in for multiple platforms (GitHub, Google, WeChat, Alipay, etc.).
+  - Dynamic passcode sign-in (via SMS or email).
+- Fully open-sourced, while no identity knowledge is required to use.
+  - Super easy setup (less than 1 min via GitPod, not joking), step-by-step tutorials and decent docs.
```
Comment (demonzoo) [inline README.md:36]:
```suggestion
  - Super easy setup (less than 1 min via GitPod, no joking), step-by-step tutorials and decent docs.
```

File: README.md [hunk 2]
```diff
@@ -23,14 +23,18 @@
...
+  - A full-function web admin console to manage the users, identities, and other things you need within a few clicks.
```
Comment (demonzoo) [inline README.md:37]:
```suggestion
  - A full-function web admin console to manage the users, identities, and other things you need with just a few clicks.
```

**Review by demonzoo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (gao-sun) [reply README.md:36]: will discuss offline

File: README.md (hunk 2 above)
Comment (gao-sun) [reply README.md:37]: will discuss offline

**Pull request merged**